nodes.go

```diff
@@ -242,6 +242,10 @@ func (n *Node) findStorageByContent(ctx context.Context, content string) (storag
 	}
 
 	for _, storage := range storages {
+		if storage.Enabled == 0 {
```

Comment:
This could be an easy thing to test if you wanted to add to the fixtures and learn to write a test on this project, not gonna hold you to it and will merge as is but if motivated open another pr with a test